### PR TITLE
Moves all of the code out of the whitesands folder

### PR DESCRIPTION
**Pull request by Bokkiewokkie**
## About The Pull Request

Every code file has been integrated into the main code now. Someone else can move the assets out to finally land the killing blow on the whitesands folder later.

Things that were deleted:
__DEFINES\DNA.dm, empty file
disease\coronavirus.dm: COVID-19 disease
datums\ai_laws.dm: anti-covid lawset and 'frank' lawset which is just AI freedom with extra steps
gamemodes\changeling\changeling.dm: `is_changeling()` proc, not used
machinery\suit_storage_unit.dm: a chief engineer SSU which already existed
sleeper party pod circuitboard removed, not used

Also updates caniuse-lite in tgui\yarn.lock

## Why It's Good For The Game

Scattered code bad, organization good.

## Changelog
:cl:
del: Removed COVID-19 disease
del: Removed 'frank' and anti-COVID lawsets
code: Moved all of the code out of the whitesands folder
/:cl:

### CODE REVIEW

**Comment by Bokkiewokkie** [comment]
> This pull request has conflicts, please resolve those before we can evaluate the pull request.

😰 the first of many

**Comment by ZephyrTFA** [comment]
So, this PR, while doing something that needs to be done, is also incredibly far too large. I'm going to slap the atomization needed label on it, it's up to Mark on whether or not the label is enforced however

**Comment by Ms-Mee** [comment]
Now all of China knows you're big

**Comment by Bokkiewokkie** [comment]
> So, this PR, while doing something that needs to be done, is also incredibly far too large. I'm going to slap the atomization needed label on it, it's up to Mark on whether or not the label is enforced however

I can give you a 'tour' of the PR in a git stream if that's easier?
I've already documented all the actual changes I've made in the body, the rest is solely moving files around.

Splitting it up only feels like it'll drag out merge conflicts for longer. Guess the decision is up to mark in the end anyways.

**Comment by MarkSuckerberg** [comment]
I'm honestly up to rip the bandage off quickly. I'll be able to help resolve any conflicts that may arise in other PRs, and I'll have a cursory review of this when I can.

File: code/modules/uplink/uplink_items.dm [hunk 1]
```diff
@@ -1978,3 +1978,114 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	limited_stock = 1
 	cant_discount = TRUE
 	include_modes = list(/datum/game_mode/nuclear)
+
+//WS additions, needs to be sorted
```
Comment (Vexylius) [inline code/modules/uplink/uplink_items.dm]: sort plz

**Review by Vexylius — APPROVED** [review]
All looks legit, really good job there

**Comment by Vexylius** [comment]
...but please do address those things I've DMd you about, then we will be testmerging it

File: code/modules/uplink/uplink_items.dm (hunk 1 above)
Comment (Bokkiewokkie) [reply code/modules/uplink/uplink_items.dm]: :tragedy:

**Review by Vexylius — APPROVED** [review]
looks legit

**Comment by rye-rice** [comment]
real

**Pull request merged**